packages/client/hmi-client/src/components/model/petrinet/tera-model-equation.vue

```diff
@@ -89,6 +95,19 @@ watch(
 	() => props.model.semantics,
 	async (newSemantics, oldSemantics) => {
 		if (isEqual(newSemantics, oldSemantics)) return;
+
+		// If there are too many states, don't bother generating the equations - Jan 2025
+		let eqLength = 0;
+		eqLength += props.model.model.states.length;
+		if (props.model.semantics?.ode.observables) {
+			eqLength += props.model.semantics.ode.observables.length;
+		}
+		if (eqLength > 250) {
```

Comment:
Very big nitpick but while I see it:
I prefer calling this 250 a var for readability and incase of re-use

```suggestion
    const equationThreshold = 250;
		if (eqLength > equationThreshold) {
```

Reply:
Done, and i also lowered the threshold to 100.